pkg/manager/member/tikv_member_manager.go

```diff
@@ -372,6 +374,29 @@ func getNewTiKVSetForTidbCluster(tc *v1alpha1.TidbCluster, cm *corev1.ConfigMap)
 		}
 	}
 
+	var additionalVolumeClaims []corev1.PersistentVolumeClaim
+	if len(tc.Spec.TiKV.StorageVolumes) > 0 {
+		for _, storageVolume := range tc.Spec.TiKV.StorageVolumes {
+			storageRequest, err := controller.ParseStorageRequest(corev1.ResourceList{
+				corev1.ResourceStorage: resource.MustParse(storageVolume.StorageSize),
```

Comment:
We should not use `MustParse` here which will cause tidb-controller-manager panic with invalid configuration.
Try to use `resource.ParseQuantity(storageSize)`, btw, I think we can call this function to validate the config in `validation.go`.

Reply:
This comment is not addressed.